db/seeds.rb

```diff
@@ -1,5 +1,6 @@
 files = [
   'availabilities',
+  'target_groups'
```

Comment:
would nice if we can make this alphabetical list so it would be easier to find something when it grows bigger